problems/countThrees.js

```diff
@@ -1,20 +1,26 @@
 /**
-* Counts the number of threes in a given number.
-* @param {number} n - The input number.
-* @returns {number} - Returns how many threes are in the input.
-*
-* ex: countThrees(300)
-* returns: 1
-*
-* ex: countThrees(3203)
-* returns: 2
-*
-* ex: countThrees(47)
-* returns: 0
-*/
-
-function countThrees() {
+ * Counts the number of threes in a given number.
+ * @param {number} n - The input number.
+ * @returns {number} - Returns how many threes are in the input.
+ *
+ * ex: countThrees(300)
+ * returns: 1
+ *
+ * ex: countThrees(3203)
+ * returns: 2
+ *
+ * ex: countThrees(47)
+ * returns: 0
+ */
 
+function countThrees(n) {
+    let counts = {}
```

Comment:
Instead of using a counter to track all elements we could just keep track of a count for one element and initialize that count at 0, and increment each time we see a 3.